### PR TITLE
Feat(client):add packages tab

### PR DESCRIPTION
**Pull request by morhag90**
Close: https://github.com/amplication/private-issues/issues/212

## PR Details

Add packages tab. 

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+type Props = AppRouteProps & {
+  match: match<{
+    resource: string;
+  }>;
+};
+
+const TITLE = "Installed Packages";
+const SUB_TITLE = "Manage your installed packages";
+
+const InstalledPackages: React.FC<Props> = ({ match }: Props) => {
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx:20]:
```suggestion
const PackagesPage: React.FC<Props> = ({ match }: Props) => {
```

Comment (yuval-hazaz) [reply packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx:20]: also rename the file

File: packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import { PackageListItem } from "./PackageListItem";
+
+type Props = AppRouteProps & {
+  match: match<{
+    resource: string;
+  }>;
+};
+
+const TITLE = "Installed Packages";
+const SUB_TITLE = "Manage your installed packages";
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx:18]: remove the words "Installed"

File: packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx (hunk 1 above)
Comment (mulygottlieb) [reply packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx:20]: @morhag90 this comment is still open (the name of the entire component + name of the file itself)

**Comment by mulygottlieb** [comment]
@morhag90 I see this PR is targeting next - I think we want to stay in an umbrella branch for now

File: packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx (hunk 1 above)
Comment (morhag90) [reply packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx:20]: @yuval-hazaz @mulygottlieb, we decided to leave it as is for now.

**Review by mulygottlieb — APPROVED** [review]

**Pull request merged**